spec/system/admin/adding_to_a_cross_core_order_spec.rb

```diff
@@ -219,6 +219,8 @@
         expect(page).not_to have_content("Cross Core Project ID")
         expect(page.has_selector?("option", text: cross_core_product_facility.name, visible: false)).to be(true)
         expect(page.has_selector?("option", text: product.name, visible: false)).to be(true)
+
+        # I think we can comment this spec, as it isn't clear why it fails from time to time.
```

Comment:
```suggestion
        # TODO: I think we can comment this spec, as it isn't clear why it fails from time to time.
```

Reply:
Oh, I wasn't clear here. I meant this as a comment for the PR to address. Are we fine commenting (or deleting) that line?